### ISSUE TITLE
Processes publish requests in parallel.

### ISSUE THREAD

**Issue by Sh031224** [figma/code-connect#82]
Describe the feature you'd like:

Our system consists of many icons and components.

In this case, I received the following response from the cli command.

```
Failed to upload to Figma (ERR_BAD_REQUEST): 413 undefined
```

I don't know the maximum number of components that can be processed, but I hope it will be possible to publish a large number of components by processing them in parallel.

**Comment by rrabherufigma** [comment]
Hi @Sh031224 , to unblock for the time being, you can either split the publishing by running it on separate directories with the --dir flag or do similar in the config.json

**Comment by caseybaggz** [comment]
Same. We tried publishing our Icon lib (2k or so icons) and even after chunking out the icons in groups, the `--dir` flag still errored. I had to manually update our `includes` to use 3 files at a time which was br000tal. Very poor DX.

Thankfully, we will rarely ever publish changes to our icon lib - but I have an overwhelming amount of sympathy for future users having to deal with this problem. 

At the very least, the CLI should allow a file argument so people can at least loop through and publish each file separately. 👈 

Even better, Code Connect should already being doing this on behalf of the users so this issue rarely exists.

1. Gather all the files (like it does)
2. Validate (like it does)
3. Loop through and individually publish each file separately (fixes issue outside of giant files longer than 13k lines of code)
